behave/README.md

```diff
@@ -22,8 +22,32 @@ to build locally:
 bash into the container and run them from there:
 `docker-compose run --entrypoint /bin/bash cla-end-to-end`
 
-## Using your local chrome browser [optional]
-If you want to see the tests running in your hosts machines chrome browser and still have the applications 
+## Current state of affairs Apple Silicon (arm64) users
+
+Apples new line of computers (Apple Silicon) no longer use Intel processors. Instead, Apple has made their own, (at the time of writing) it isn't widely supported. 
+
+To run the tests locally just run this script for Apple Silicon users. 
+
+`./run_test_local_m1.sh`
+
+**What are the known problems for Apple Silicon users?**
+
+Currently, `phantom.js` is used for unit testing in cla_frontend and does not support `arm64` which causes `uwsgi` to fail to start.
+
+`events.js` within `cla_public`, throws an unhandled error 'qemu-i386', this is because '/lib/ld-linux.so.2' directory does not exist in `arm64` platforms.
+
+Selenium Chrome also does not support `arm64`. In order to get it to work use this docker image: `seleniarm/standalone-chromium:4.0.0-beta-1-20210215`
+[seleniarm](https://github.com/SeleniumHQ/docker-selenium#experimental-mult-arch-aarch64armhfamd64-images)
+
+`cla_frontend` and `cla_public` will fail to build if you do not add the required platform architecture values. e.g. `platform: linux/arm64` within a docker-compose.yml file or build arguments for Docker.
```

Comment:
> `cla_frontend` and `cla_public` will fail to build if you do not add the required platform architecture values. e.g. `platform: linux/arm64` within a docker-compose.yml file or build arguments for Docker.

Is this suggesting editing the docker-compose.yml file locally?
If so can these changes be put into the `behave/docker-compose.m1.yml` like frontend application?

Reply:
Ah okay, I can see now how that is a bit confusing. So I was trying to document the reasons why the platform has to be specified in the `behave/docker-compose.m1.yml` / `cla_end-to-end` repo. 

Now looking at it. It looks like I'm giving instructions on how to run `cla_frontend` etc locally, which should be documented in `cla_frontend` READ.ME. 

I'll make a change to this sentence to make it clearer I'm just talking about why it's needed in the `behave/docker-compose.m1.yml`